app/Http/Controllers/Staff/CourseRevisionController.php

```diff
@@ -57,6 +57,8 @@ public function store(Request $request, Course $course)
      */
     public function destroy($course, CourseRevision $revision)
     {
+        $this->authorize($revision);
```

Comment:
no action?? I think it should be `$this->authorize('destroy', $revision)`

Reply:
It worked this way as well. Can change if required for better understanding

Reply:
Yes, being explicit is better.